### ISSUE TITLE
Beacon SSIDs not showing up

### ISSUE THREAD

**Issue by dotada** [spacehuhn/esp8266_deauther#1654]
> Have you searched for existing (open and closed) issues describing the same problem?

Yes

**Describe the error**
Not sure if I'm doing something wrong but when launching a beacon attack with any SSIDs sometimes some of the SSIDs show up, sometimes they don't. I noticed that V3 doesn't have this issue although the web interface is kind of essential for me.

**To Reproduce**
Launch a beacon attack with more than 3 SSIDs and monitor which SSIDs show up

**Expected behavior**
All SSIDs are discovered.

**Screenshots**
Not sure I can add any here.

**Environment (please complete the following information):**
 - OS: Tested on Windows 11 and Android
 - Browser Opera GX
 - Version 2.0
 - Hardware NodeMCU esp8266 board

**Attempts**
Change up beacon related settings and modifying the beacon attack source code.

**Additional context**
Long story short, when launching a beacon attack on V2 1 to 3 of the SSIDs will show up at max instead of all of them whereas on V3 all SSIDs show up as intended.
